README.md

```diff
@@ -175,49 +175,77 @@ There are several data loading implementations of popular datasets across differ
 
 ## Frequently Asked Questions (FAQ)
 
-Q: What should I do if the existing set of DataPipes does not do what I need?
+<details>
+<summary>
+What should I do if the existing set of DataPipes does not do what I need?
+</summary>
 
-A: You can
+You can
 [implement your own custom DataPipe](https://pytorch.org/data/main/tutorial.html#implementing-a-custom-datapipe). If you
 believe your use case is common enough such that the community can benefit from having your custom DataPipe added to
 this library, feel free to open a GitHub issue. We will be happy to discuss!
+</details>
 
-Q: What happens when the `Shuffler` DataPipe is used with DataLoader?
+<details>
+<summary>
+What happens when the <code>Shuffler</code> DataPipe is used with DataLoader?
+</summary>
 
-A. In order to enable shuffling, you need to add a `Shuffler` to your DataPipe line. Then, by default, shuffling will
+In order to enable shuffling, you need to add a <code>Shuffler</code> to your DataPipe line. Then, by default, shuffling will
 happen at the point where you specified as long as you do not set `shuffle=False` within DataLoader.
+</details>
 
-Q: What happens when the `Batcher` DataPipe is used with DataLoader?
+<details>
+<summary>
+What happens when the <code>Batcher</code> DataPipe is used with DataLoader?
+</summary>
 
-A: If you choose to use `Batcher` while setting `batch_size > 1` for DataLoader, your samples will be batched more than
+If you choose to use <code>Batcher</code> while setting <code>batch_size > 1</code> for DataLoader, your samples will be batched more than
```

Comment:
If this only applies to summary, let's change this part. Then this PR is good to go.

Reply:
Good call, new commit added.